api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -708,6 +709,13 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     dbWorkspace = workspaceService.updateUserRoles(dbWorkspace, dbUserRoles);
     ShareWorkspaceResponse resp = new ShareWorkspaceResponse();
     resp.setWorkspaceEtag(Etags.fromVersion(dbWorkspace.getVersion()));
+    List<UserRole> updatedUserRoles = dbWorkspace.getWorkspaceUserRoles()
+        .stream()
+        .map(r -> new UserRole().email(r.getUser().getEmail()).role(r.getRole()))
+        // Reverse sorting arranges the role list in a logical order - owners first, then by email.
```

Comment:
Based on the wording, this is what I'm assuming but double-checking: we aren't depending on this ordering in any way right? This is purely for display niceness?

Reply:
That's exactly right, it's purely for display. When adding and removing users, the UI replaces the old list of roles with the new list of roles. If we don't default sort in some way (either here in the java, or in the typescript), the order bounces around between adds/removes.